js/extension/features/block-sellers.js

```diff
@@ -30,30 +30,43 @@ rl.ready(() => {
    *
    * @method blockSellers
    * @param {String} type Either 'hide' or 'tag'
+   * @param {String} querySelector Either 'marketplaceQuerySelector' or 'wantlistMessageQuerySelector' const
+   * @param {String} tag Either 'a' (anchor) or 'strong' tag
+   * @param {String} clazz Either 'shortcut_navigable' (marketplace) or 'wantlist-card' (wantlist message)
    * @return {function}
    */
 
-  window.blockSellers = function blockSellers(type) {
+  window.blockSellers = function blockSellers(type, querySelector) {
 
-    let _class = type === 'hide' ? 'hidden-seller' : 'blocked-seller';
+    let _class = type === 'hide' ? 'hidden-seller' : 'blocked-seller',
+        tag,
+        clazz;
+
+    if (querySelector === marketplaceQuerySelector) {
+      tag = 'a';
```

Comment:
This should actually be the `li` element, not the `a` element. It causes a visual alignment issue when the `span` is inserted into the `a`.

Reply:
Thanks, that was maybe c&p error. Now it should be fine.